packages/governance/src/multisig/multisig.cairo

```diff
@@ -0,0 +1,447 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.18.0 (governance/multisig/multisig.cairo)
+
+/// # Multisig Component
+#[starknet::component]
+pub mod MultisigComponent {
+    use core::hash::{HashStateTrait, HashStateExTrait};
+    use core::num::traits::Zero;
+    use core::panic_with_felt252;
+    use core::pedersen::PedersenTrait;
+    use crate::multisig::interface::{IMultisig, TransactionID, TransactionState};
+    use crate::timelock::utils::call_impls::{HashCallImpl, HashCallsImpl, CallPartialEq};
+    use starknet::account::Call;
+    use starknet::storage::{Map, StorageMapReadAccess, StorageMapWriteAccess};
+    use starknet::storage::{StoragePointerReadAccess, StoragePointerWriteAccess};
+    use starknet::syscalls::call_contract_syscall;
+    use starknet::{ContractAddress, SyscallResultTrait};
+    use starknet::{get_caller_address, get_contract_address, get_block_number};
+
+    #[storage]
+    pub struct Storage {
+        pub Multisig_quorum: u8,
+        pub Multisig_signers_count: u32,
```

Comment:
This can be part of the same struct implementing StorePacking to save both values into one felt to save storage.

Reply:
Introduced a `SignersInfo` struct and implemented `StorePacking` for it